package.json

```diff
@@ -76,13 +76,17 @@
     "renovate-config-seek": "0.4.0",
     "surge": "^0.23.1",
     "tsx": "^3.12.1",
-    "typescript": "^4.5.0"
+    "typescript": "~5.1.6"
   },
   "volta": {
     "node": "18.15.0"
   },
   "packageManager": "pnpm@7.30.0",
   "pnpm": {
+    "overrides": {
+      "@braid-design-system/generate-component-docs": "workspace:*",
+      "braid-design-system": "workspace:*"
+    },
```

Comment:
Had to do these shenanigans until `manypkg` supports the `workspace:` protocol https://github.com/Thinkmill/manypkg/pull/186